src/Microsoft.Fx.Portability/BreakingChange.cs

```diff
@@ -28,7 +28,8 @@ public BreakingChange DeepCopy()
                 SourceAnalyzerStatus = this.SourceAnalyzerStatus,
                 BugLink = this.BugLink,
                 Notes = this.Notes,
-                ImpactScope = this.ImpactScope
+                ImpactScope = this.ImpactScope,
```

Comment:
No, we were setting it previously. The only change to this line is to add the , at the end since it's no longer the last member being initialized.